examples.py/3D/Form/bricktower.py

```diff
@@ -15,35 +15,36 @@
 def setup():
     size(640, 360, OPENGL)
 
+
 def draw():
-  background(0)
-  (tempX, tempY, tempZ) = (0, 0, 0)
-  fill(182, 62, 29)
-  noStroke()
-  # Add basic light setup
-  lights()
-  translate(width / 2, height * 1.2, -380)
-  # Tip tower to see inside
-  rotateX(radians(-45))
-  # Slowly rotate tower
-  rotateY(frameCount * PI / 600)
-  for i in range(brickLayers):
+    background(0)
+    (tempX, tempY, tempZ) = (0, 0, 0)
```

Comment:
> ... we're here to teach and to communicate clearly...

OK, I concede Jython is a :snail:. But do you really believe `(tempX, tempY, tempZ) = (0, 0, 0)`  communicates the idea of assigning to `0` better than `tempX = tempY = tempZ = 0`?  :confused: